### DISCUSSION TITLE
Add new fields to the Reference Work table

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5818] [Feature Requests]
> We would like to request the addition of a number of spare text fields to the Reference Work table when you update the schema the next time. We are attempting to migrate some existing data for the insect collection but are running out of available text fields.

**Requested By:** Corinna P at CSIRO ([Asana](https://app.asana.com/0/1201972579963460/1204674079858696/f))

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8202
